src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -671,3 +715,21 @@ function New-AzConnectedKubernetes {
         Return $Response
     }
 }
+
+function Test-ConfigurationSyntax {
```

Comment:
We need to make sure that we move this out of this file and/or explicitly hide it otherwise it gets exposed by `autorest`.